src/main/webapp/testFeed.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <link rel="stylesheet" href="feedStyle.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+    <link href="https://unpkg.com/material-components-web@latest/dist/material-components-web.min.css" rel="stylesheet">
+    <script src="https://unpkg.com/material-components-web@latest/dist/material-components-web.min.js"></script>
+    <script src="testScript.js"></script>
+  </head>
+  <body>
+    <div class = "wrap">
+        <div class = "navigation_bar" id="nav_bar"></div>
+        <div class = "body_main" id= "body_main"></div>
+            <div class = "left_col" id = "left_col"></div>
+            <div class = "mid_col" id = "mid_col">
+                <!-- this form will be used to add new comments once the new_comment endpoint is done -->
+                <!-- <form action="/new_comment" method="POST">
+                    <input type="text" name="comment">
+                    <button>Submit</button>
+                </form> -->
```

Comment:
delete commented code

Reply:
I have it commented out so I remember to implement it when I can.

Reply:
Ah okay